src/twisted/python/compat.py

```diff
@@ -829,6 +830,34 @@ def _bytesRepr(bytestring):
 
 
 
+def get_async_param(isAsync=None, **kwargs):
```

Comment:
I think that this should be private...as in, it should only be used inside Twisted core and not used by other projects... so this should be `def _get_async_param`